core/src/main/scala/kafka/utils/CommandLineUtils.scala

```diff
@@ -36,16 +37,27 @@ object CommandLineUtils extends Logging {
     return commandOpts.args.length == 0 || commandOpts.options.has(commandOpts.helpOpt)
   }
 
+  def isPrintVersionNeeded(commandOpts: CommandDefaultOptions): Boolean = {
+    return commandOpts.options.has(commandOpts.versionOpt)
```

Comment:
nit: return is implicit in scala

Reply:
Just can't get used to that :)
I'll fix it.

Reply:
Don't forget this 🙂

Reply:
I indeed forgot :)
Done now.
I've also opened #6629 to remove the existing return in the code and not "pollute" this PR with that.